go.mod

```diff
@@ -1,9 +1,12 @@
 module github.com/hashicorp/vault-plugin-secrets-azure
 
-go 1.19
+go 1.18
```

Comment:
Should this be downgraded?

Reply:
I had to downgrade it to run go mod tidy, will switch it back 👍